config/initializers/nationalities.rb

```diff
@@ -269,3 +269,6 @@
   'SE',
   'GB'
 ].freeze
+
+PROVISIONALLY_ELIGIBLE_FOR_GOV_FUNDING_COUNTRY_CODES = %w[GB IE].freeze
```

Comment:
I think we might need to include Isle of Man here as well country code:`IM` (taken from https://docs.google.com/document/d/1dQhlL45SlQX9YiI3jDX0T5u1GUWfjb9uYQYYh6rE0Lk/edit#) unless `IM` is referring to something else, although I can't see it in the dropdown from the FE 🤔

Reply:
I was slightly confused by this. EU_COUNTRY_CODES includes some extra codes (AX FO GF GI IM) which are not included in the NATIONALITIES array. But I don't think we need to worry about this because when a candidate enters nationalities we validate that they are in the NATIONALITY array so `IM` etc cannot be saved.

Reply:
Will we spare ourselves future surprises if we include the proper rule here (ie incl `IM` et al), just in case the front end does change?

Reply:
makes sense. I need to read up on this 😅 
Actually, is our logic correct? GOV.UK says

> You can apply for full support if **all** the following apply:
> 
> - you’re a UK or Irish national or have ‘settled status’ (no restrictions on how long you can stay)
> - you normally live in England
> - you’ve been living in the UK, the Channel Islands or the Isle of Man for 3 continuous years before the first day of your course, apart from temporary absences such as going on holiday
https://www.gov.uk/student-finance/who-qualifies#when-youre-eligible-for-full-support

Do we need to be checking domicile etc for UK citizens as well?

Reply:
Reluctant to re-open this can of worms now, but worth double checking with James and Matilde that we're acknowledging this slight fudge